docs/cloud/manage/backups/export-backups-to-own-cloud-account.md

```diff
@@ -57,13 +61,19 @@ You will need the following details to export/restore backups to your own CSP st
 2. Access HMAC key and HMAC secret.
 
 <hr/>
-# Backup / Restore
 
-## Backup / Restore to AWS S3 Bucket {#backup--restore-to-aws-s3-bucket}
+# Backup / restore {#backup-restore}
+
+:::note
+1. For restoring the backup from your own bucket into a new service, you will need to update the trust policy of your backups storage bucket to allow access from the new service.
+2. The Backup / Restore commands need to be run from the database command line. For restore to a new service, you will first need to create the service and then run the command.   
+:::
 
-### Take a DB Backup {#take-a-db-backup}
+## Backup / restore to AWS S3 bucket {#backup--restore-to-aws-s3-bucket}
 
-**Full Backup**
+### Take a DB backup {#take-a-db-backup}
+
+**Full backup**
 
```

Comment:
We should move up the recommended command, it should be first, otherwise it could be missed